profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -863,6 +856,39 @@ function tmgmt_poetry_form_tmgmt_entity_ui_translate_form_alter(&$form, &$form_s
   }
 }
 
+/**
+ * Check if 'Request translation' is possible.
+ *
+ * @param array $form_state
+ *   The $form_state variable from form.
+ *
+ * @return bool
+ *   Returns true if 'Request translation' is possible, false otherwise.
+ */
+function _tmgmt_poetry_translations_possible(array $form_state) {
+
+  $translations_possible = array();
+
+  // Checking if we have a node with the appropriate moderation state.
+  // Getting the moderation states which allows to request the translation.
+  $states = array_values(_tmgmt_workbench_available_states());
+  $state = $form_state['entity']->workbench_moderation['current']->state;
+
+  if (in_array($state, $states, 'strict')) {
+    $translations_possible = array('regular');
+  }
+
+  // If Cart translation is possible, Regular translation shouldn't be possible.
```

Comment:
Here we assume the existence of a cart module and act accordingly, but that is not the case. Part of this functionality should go into the cart module.
It should be the cart modules responsibility to show or hide its button.

Reply:
It's actually what's happening. Cart module will add it's own button.

But in the situation when cart module adds it's own button, we don't want to show the regular poetry translation button. So Poetry module needs to check that here.